include/openenclave/bits/result.h

```diff
@@ -377,6 +377,11 @@ typedef enum _oe_result
      */
     OE_SGX_QUOTE_LIBRARY_ERROR,
 
+    /**
+     * The FS register was in the bad state.
+     */
+    OE_SGX_BAD_FS,
```

Comment:
Avoid changes to public interface until absolutely needed.

Reply:
reverted.